ontopy/ontodoc.py

```diff
@@ -392,23 +392,30 @@ def itemdoc(
         # Instances (individuals)
         if hasattr(item, "instances"):
             points = []
-            for entity in [
-                _ for _ in item.instances() if item in _.is_instance_of
-            ]:
-                points.append(
-                    point_style.format(
-                        point=asstring(entity, link_style), ontology=onto
-                    )
-                )
-            if points:
-                value = points_style.format(
-                    points="".join(points), ontology=onto
-                )
-                doc.append(
-                    annotation_style.format(
-                        key="Individuals", value=value, ontology=onto
+            for instance in item.instances():
+                try:
+                    if item in instance.is_instance_of:
+                        points.append(
+                            point_style.format(
+                                point=asstring(instance, link_style),
+                                ontology=onto,
+                            )
+                        )
+                    if points:
+                        value = points_style.format(
+                            points="".join(points), ontology=onto
+                        )
+                        doc.append(
+                            annotation_style.format(
+                                key="Individuals", value=value, ontology=onto
+                            )
+                        )
+                except TypeError:
+                    warnings.warn(
```

Comment:
Very good warning message. Are we sure that there are not other TypeErrors what will be catched by this except statement?

Reply:
No, I am not sure that there are not others. But I think it is only punning that will be caught by this because we ask for an indivudal, and then we iterate of the concepts that the individual is an instance of. But I think this is a good point. If there is punning I know that is_instance_of returns 'property' and not a list of concepts, so I changed it to be a positive test for that. Better?